pkgs/applications/audio/mopidy/default.nix

```diff
@@ -7,6 +7,8 @@ lib.makeScope newScope (self: with self; {
 
   mopidy = callPackage ./mopidy.nix { };
 
+  mopidy-argos = callPackage ./argos.nix { };
```

Comment:
If this doesn't depend on mopidy directly, put in `pkgs/by-name`. See doc there for instructions.

Reply:
Argos only supports connecting to a local Mopidy installation and thus requires Mopidy to be installed.

Reply:
It does not directly depend on the package in the scope of mopidy, and the probability that other packages in the scope of mopidy depend on it is very low.

This scope plays this role: when callPackage is used for the packages in the scope, other packages will be expanded as part of the input. If there is a package with the same name in the top-level, it will be overwritten. In this way, the scope can be maintained as a whole, allowing multiple versions of the scope to coexist, for example.

Reply:
The reason I say this is that scopes are more complex to deal with especially when it comes to overriding. Top-level by-name packages are preferred.

Reply:
That's good to know, thank you for explaining!